docs/react/week-1/lesson.md

```diff
@@ -5,6 +5,9 @@ sidebar_label: Lesson
 ---
 
 import Feedback from "@theme/Feedback";
+import { ExerciseDDemo } from './demos/ExerciseD.js'
```

Comment:
Could we add a comment here explaining how this works and linking away to other documentation?

Reply:
Good point, added in 98106f0